lib/model.js

```diff
@@ -4091,14 +4038,28 @@ function _update(model, op, conditions, doc, options, callback) {
  *        console.log(docs);
  *     }).then(null, handleError).end()
  *
- * @param {Object} o an object specifying map-reduce options
+ * @param {Object} opts an object specifying map-reduce options
+ * @param {Boolean} [opts.verbose=false] provide statistics on job execution time
+ * @param {String} [opts.readPreference]
```

Comment:
`[ReadPreference|String]` since can also accept a MongoDB ReadPreference instance

Reply:
should be done in bb19246d7ec10c56418cf715b15d04ddbe7a162b